src/utils/attributes/attributes.ts

```diff
@@ -30,7 +30,7 @@ export const getAttributeContent = (contentType: AttributeContentType, content:
          case AttributeContentType.Integer:
          case AttributeContentType.String:
          case AttributeContentType.Text:
-            return content.data.toString();
+            return content?.data?.toString();
```

Comment:
this seems to be wrong, content.data should be always available for those contentTypes and if not, then it's problem on BE and it should fail in FE